understanding/focus-appearance.html

```diff
@@ -445,498 +372,379 @@ <h4>Inline links</h4>
                   </p>
                   
                   
-                  <p>Result: the focus indicator for the tab item must meet the requirements of this Success
-                     Criterion, judging focus with both selected and unselected tab items.
+                  <p>結果: タブ項目のフォーカスインジケータは、選択されたタブ項目と選択されていないタブ項目の両方でフォーカスを判断するという、この達成基準の要件を満たさなければならない。
                   </p>
                   
                   
-                  <p>A <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#example"> functional example of sub-component-only tab focus</a> has an indicator that is large enough (at least four times the shortest side) with
-                     sufficient contrast to pass the focus area language of this Success Criterion.
+                  <p><a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#example">サブコンポーネントだけのタブフォーカスの機能的な例</a>は、この達成基準のフォーカス領域の言語に合格するのに十分な大きさ (少なくとも最短辺の 4 倍以上) 及び十分なコントラストのインジケータがある。
                   </p>
                   
                   
-                  <h3>Where something with focus is not a user interface component</h3>
+                  <h3>フォーカスを持つものがユーザインタフェース コンポーネントではない場合</h3>
                   
-                  <p>Some pages contain very large editing regions, such as web implementations of word
-                     processors and code editors. Unlike a <code>textarea</code> element, which is a user interface component, these large editing regions do not
-                     typically meet the definition of <a href="https://w3c.github.io/wcag/guidelines/22/#dfn-user-interface-components">user interface components</a>; they are not "perceived by users as a single control for a distinct function." Providing
-                     focus indicators around such editing regions may still be beneficial to some; however,
-                     where the region is not perceived as a single control, it is not covered by this Success
-                     Criterion. The web page will still need to provide a insertion point (caret indicator)
-                     in such editing regions in order to meet the requirements of <a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html">2.4.7 Focus Visible</a>.
+                  <p>一部のページには、ワードプロセッサ、コードエディタのウェブ実装など、非常に大きな編集領域を含むものがある。ユーザインタフェース コンポーネントである <code>textarea</code> 要素とは異なり、これらの大きな編集領域は一般的に、<a href="https://waic.jp/translations/WCAG22/#dfn-user-interface-components">ユーザインタフェース コンポーネント</a>の定義を満たさない。つまり、「利用者が個別の機能の単一のコントロールとして認識する」ことはない。そのような編集領域の周りにフォーカスインジケータを表示することは、一部の人にとっては有益かもしれないが、領域が単一のコントロールとして認識されない場合は、この達成基準の対象にはならない。ウェブページは、<a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html">2.4.7 フォーカスの可視化</a>の要件を満たすために、そのような編集領域に挿入位置 (キャレットインジケータ) を提供する必要がある。
```

Comment:
用語「ユーザインタフェース コンポーネント」の定義では、distinct function を「特定の機能」と訳しているので、それに合わせたいと思います。

```suggestion
                  <p>一部のページには、ワードプロセッサ、コードエディタのウェブ実装など、非常に大きな編集領域を含むものがある。ユーザインタフェース コンポーネントである <code>textarea</code> 要素とは異なり、これらの大きな編集領域は一般的に、<a href="https://waic.jp/translations/WCAG22/#dfn-user-interface-components">ユーザインタフェース コンポーネント</a>の定義を満たさない。つまり、「利用者が特定の機能の単一のコントロールとして認識する」ことはない。そのような編集領域の周りにフォーカスインジケータを表示することは、一部の人にとっては有益かもしれないが、領域が単一のコントロールとして認識されない場合は、この達成基準の対象にはならない。ウェブページは、<a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html">2.4.7 フォーカスの可視化</a>の要件を満たすために、そのような編集領域に挿入位置 (キャレットインジケータ) を提供する必要がある。
```